app/src/Command/LDAPImportCommand.php

```diff
@@ -131,6 +131,9 @@ private function importUsers(): void
                 $listAliases = array_unique($listAliases);
 
                 foreach ($listAliases as $aliasEmail) {
+                    if (!filter_var($aliasEmail, FILTER_VALIDATE_EMAIL)) { // ignore invalid emails (#286)
```

Comment:
Le code se suffit à lui-même, pas besoin du commentaire :

```suggestion
                    if (!filter_var($aliasEmail, FILTER_VALIDATE_EMAIL)) {
```